### DISCUSSION TITLE
Collections

### DISCUSSION THREAD

**Discussion by JPH71** [CodyBerenson/PGMA-Modernized#314] [Ideas]
As Film Genres are now listed under Categories, I suggest we remove genres from Collections....

**Comment by fourstix** [comment]
I think that's a good idea.  I don't use Collections for Genres since they are available under categories.

**Reply by JPH71** [reply]
I am also coming to the conclusion that genres as categories better... And we keep collections for Actors, Directors and actual collections (Series of Films)

**Reply by jfreiman** [reply]
I'm fine with that too.
I personally force my Genre's in mp4 metadata to a very limited selection to keep it simple (KISS) and allow the Categories to go hog wild.

FYI, The Genre's I have chosen to use are:

- Classic (used for all videos prior to 2001)
- Pre-Condom (always used along with Bareback and Classic) "Pre-Condom/Bareback/Classic"
- Bareback
- Safe Sex
- Solo
- Non-Sex
- BDSM
- Masturbation
- Oral Sex
- Documentary
- Animated

Approx every week I go through and remove/delete all the Genres added through PGMA sources.

I find it's much easier to find, narrow down what you want to watch this way -- especially with the Plex TV interface.

**Comment by JPH71** [comment]
Or you could just set your user gay tidy to do that automatically for you..
that was what it was designed for...
